tests/tests.js

```diff
@@ -151,25 +151,11 @@ beforeEach: function() {
     // Initialize a fresh checkit object and a dummy active calendar for every test
     this.$fixture = $('#qunit-fixture');
     this.$calendarDiv = this.$fixture.find('#calendarDiv');
-    this.$fixture.append(
-        `<script>
-        $(document).ready( function() {
-        checkit = new CheckIt('localStorage', $('#qunit-fixture #calendarDiv'));
-        });
-        </script>`);
  
-    this.checkit = checkit;
+    this.checkit = new CheckIt('localStorage', $('#qunit-fixture #calendarDiv'));
```

Comment:
On that note, it's probably a good idea to log a warning during initialization if checkit is initialized with a selector that doesn't select anything.